spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -512,7 +512,7 @@ def upload_trailer_preview(app_version, upload_trailer_preview)
       raise "app_version is required" unless app_version
       raise "upload_trailer_preview is required" unless upload_trailer_preview
 
-      du_client.upload_trailer_preview(app_version, upload_trailer_preview, content_provider_id, sso_token_for_image)
+      du_client.upload_video_preview(app_version, upload_trailer_preview, content_provider_id, sso_token_for_image)
```

Comment:
I really don't like the fact that we have both video_preview and trailer_preview in the code...